Modules/Microsoft365DSC/DSCResources/MSFT_EXOHostedContentFilterPolicy/readme.md

```diff
@@ -63,6 +63,7 @@ EXOHostedContentFilterPolicy TestHostedContentFilterPolicy {
   LanguageBlockList                        = @('AF', 'SQ', 'AR', 'CY', 'YI')
   RegionBlockList                          = @('AF', 'AX', 'AL', 'DZ', 'ZW')
   HighConfidenceSpamAction                 = 'Quarantine'
+  HighConfidencePhishAction                = 'Quarantine'
```

Comment:
For future PRs, you don't need to add the examples to the readme.md. These are automatically pulled from the example folder and dynamically built into the wiki page.